frontend/src/bundles/talent-onboarding/store/slice.ts

```diff
@@ -27,11 +33,41 @@ const { reducer, actions, name } = createSlice({
     reducers: {},
     extraReducers(builder) {
         builder.addCase(updateTalentDetails.fulfilled, (state, action) => {
+            state.dataStatus = DataStatus.FULFILLED;
+
             for (const key in action.payload) {
                 const typedKey = key as keyof UserDetailsUpdateRequestDto;
-                state[typedKey] = action.payload[typedKey];
+                state[typedKey] =
+                    typedKey === 'englishLevel' &&
+                    action.payload[typedKey] === null
+                        ? ' '
+                        : action.payload[typedKey];
+            }
+        });
+        builder.addCase(getTalentDetails.fulfilled, (state, action) => {
+            state.dataStatus = DataStatus.FULFILLED;
+            for (const key in action.payload) {
+                const typedKey = key as keyof UserDetailsFindRequestDto;
+
+                state[typedKey] =
+                    typedKey === 'englishLevel' &&
+                    action.payload[typedKey] === null
+                        ? ' '
+                        : action.payload[typedKey];
             }
```

Comment:
Why we need to do it ?

Reply:
Hello, this is based off of an earlier version of #344. This has been addressed there in a newer version.

Reply:
Hello, this is based off of an earlier version of #344. This has been addressed there in a newer version.